src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ToolStripSplitButtonTests.cs

```diff
@@ -1,72 +1,436 @@
+    public void ToolStripSplitButton_DefaultItem_Set_GetReturnsExpected(ToolStripItem? value)
     {
-        using ToolStripSplitButton button = new()
-        {
-            DefaultItem = value
-        };
-        Assert.Same(value, button.DefaultItem);
-
-        // Set same.
-        button.DefaultItem = value;
-        Assert.Same(value, button.DefaultItem);
+        _toolStripSplitButton.DefaultItem = value;
+        _toolStripSplitButton.DefaultItem.Should().Be(value);
     }
 
     [WinFormsFact]
     public void ToolStripSplitButton_DefaultItem_SetWithHandler_CallsOnDefaultItemChanged()
     {
-        using ToolStripSplitButton button = new();
-
         int callCount = 0;
         EventHandler handler = (sender, e) =>
         {
-            Assert.Same(button, sender);
-            Assert.Same(EventArgs.Empty, e);
+            sender.Should().Be(_toolStripSplitButton);
+            e.Should().Be(EventArgs.Empty);
             callCount++;
         };
-        button.DefaultItemChanged += handler;
+        _toolStripSplitButton.DefaultItemChanged += handler;
 
         // Set non-null.
         using SubToolStripItem item1 = new();
-        button.DefaultItem = item1;
-        Assert.Same(item1, button.DefaultItem);
-        Assert.Equal(1, callCount);
+        _toolStripSplitButton.DefaultItem = item1;
+        _toolStripSplitButton.DefaultItem.Should().Be(item1);
+        callCount.Should().Be(1);
 
         // Set same.
-        button.DefaultItem = item1;
-        Assert.Same(item1, button.DefaultItem);
-        Assert.Equal(1, callCount);
+        _toolStripSplitButton.DefaultItem = item1;
+        _toolStripSplitButton.DefaultItem.Should().Be(item1);
+        callCount.Should().Be(1);
 
         // Set different.
         using SubToolStripItem item2 = new();
-        button.DefaultItem = item2;
-        Assert.Same(item2, button.DefaultItem);
-        Assert.Equal(2, callCount);
+        _toolStripSplitButton.DefaultItem = item2;
+        _toolStripSplitButton.DefaultItem.Should().Be(item2);
+        callCount.Should().Be(2);
 
         // Set null.
-        button.DefaultItem = null;
-        Assert.Null(button.DefaultItem);
-        Assert.Equal(3, callCount);
+        _toolStripSplitButton.DefaultItem = null;
+        _toolStripSplitButton.DefaultItem.Should().BeNull();
+        callCount.Should().Be(3);
 
         // Remove handler.
-        button.DefaultItemChanged -= handler;
-        button.DefaultItem = item1;
-        Assert.Equal(item1, button.DefaultItem);
-        Assert.Equal(3, callCount);
+        _toolStripSplitButton.DefaultItemChanged -= handler;
+        _toolStripSplitButton.DefaultItem = item1;
+        _toolStripSplitButton.DefaultItem.Should().Be(item1);
+        callCount.Should().Be(3);
+    }
+
+    [WinFormsFact]
+    public void ToolStripSplitButton_Ctor_Default()
+    {
+        using ToolStripSplitButton toolStripSplitButton = new();
+
+        toolStripSplitButton.Text.Should().BeEmpty();
```

Comment:
```suggestion
        _toolStripSplitButton.Text.Should().BeEmpty();
```
I don't think you have to invoke constructor because we are guaranteed that it is called, we can test properties of our field.